ledger/src/blocktree.rs

```diff
@@ -1175,6 +1175,14 @@ impl Blocktree {
             .collect()
     }
 
+    pub fn write_transaction_status(
+        &self,
+        index: (Slot, Signature),
+        status: &RpcTransactionStatus,
+    ) -> Result<()> {
+        self.transaction_status_cf.put(index, status)
```

Comment:
One thing that came to mind as I was thinking about this -- should we distinguish at
all between "put when empty" and "put/overwrite" cases? (Was wondering if we'd want
to raise an error if there's already data existing for the given key, or if "last write wins"
semantics is ok).

Reply:
Fair question. I think last-write-wins should be fine... if we are processing/committing transactions more than once, we have much bigger problems ;)